cmd/drand-cli/cli.go

```diff
@@ -559,8 +559,20 @@ func CLI() *cli.App {
 	app.Version = version.String()
 	app.Usage = "distributed randomness service"
 	// =====Commands=====
-	app.Commands = appCommands
-	app.Flags = toArray(verboseFlag, folderFlag)
+	// we need to copy the underlying commands to avoid races, cli sadly isn't supporting well concurrent executions
```

Comment:
'cli sadly doesn't support concurrent executions well' - when english goes full german word order mode ;p

Reply:
like @mcamou said earlier, I wish there was a way to tell the race condition checker to 🤐 for things like this